docs/source/push-changes.md

```diff
@@ -2,7 +2,7 @@
 
 ## Push your changes to GitHub repo
 
-Whenever you finish working on your project or you need to stop, push your changes to GitHub, so that all your work can be saved. In order to do that:
+If you are running this tutorial on Operate First or the Meteor environment, your work-in-progress notebooks should be saved in your JupyterHub PVC by hitting the save button on the top panel. Whenever you finish working on your project you can push your changes to your GitHub fork so that all your work can be saved. In order to do that from within JupyterHub you can:
```

Comment:
Will the meteor environment include a PVC? I would assume that it would not have one that would persist form session to session. Can you update this to reflect Operate First will save your work but meteor will not.

Reply:
Made the changes.